core/storage/suggestion/gae_models.py

```diff
@@ -1329,13 +1329,6 @@ class TranslationReviewStatsModel(base_models.BaseModel):
     # attributes.
     accepted_translation_word_count = datastore_services.IntegerProperty(
         required=True, indexed=True)
-    # The total word count of accepted translations with reviewer edits.
-    # Excludes HTML tags and attributes.
-    accepted_translations_with_reviewer_edits_word_count = (
```

Comment:
Why are we dropping this one?

Reply:
We do not use this parameter for any calculations to compute levels/milestones. Furthermore, this is not displayed in the [reviewer stats](https://www.figma.com/file/SNOHAatFcCvdvIAdxAzF6E/Contributor-Dashboard-Achievements?node-id=112%3A16) as well. Hence, it is removed.l